master/docs/manual/cfg-workers-openstack.rst

```diff
@@ -106,12 +106,14 @@ These are the same details set in either environment variables or passed as opti
 ``nova_args``
     (optional)
     A dict that will be appended to the arguments when creating a VM.
-    Buildbot uses the OpenStack Nova version 1.1 API by default (see client_version).
+    Buildbot uses the OpenStack Nova version 2 API by default (see client_version).
```

Comment:
Starting with novaclient 2.27.0 (released in April 2015), it will automatically use version 2 if you ask to use version 1.1. This message is printed out when creating the client:

    Version 1.1 is deprecated, using alternative version 2 instead.

I noted in the docs that 1.1 is deprecated, so I can add that you need a version older than 2.27.0 to actually use 1.1.